sefaria/model/place.py

```diff
@@ -46,8 +46,24 @@ def primary_name(self, lang=None):
 
     def secondary_names(self, lang=None):
         return self.name_group.secondary_titles(lang)
-
-    ###
+    
+    @classmethod 
+    def create_new_place(cls, en, he=None):
```

Comment:
I see cls isn't used in the entire class method, perhaps replace `p = Place().load({'key': en})` with `p = cls().load({'key': en})` so that it justifies this being a class method. Also, once that us the case, this kind of function can be abstracted to be used on multiple objects